specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/examples/event-grid-schema/Resources_Deleted.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "d4611260-d179-4f86-b196-3a9d4128be2d",
+  "topic": "/subscriptions/{subscription-id}",
+  "subject": "/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storageAccount-name}",
+  "data": {
+    "resourceInfo": {
+      "id": "/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storageAccount-name}",
+      "name": "storageAccount-name",
+      "type": "Microsoft.Storage/storageAccounts"
+    },
+    "operationalInfo": {
```

Comment:
should we extract this to its own base type such that any future updates to operationalInfo is only needed in one place?

Reply:
done